zboxcore/sdk/live_upload_reader_youtubedl.go

```diff
@@ -102,15 +101,15 @@ func (r *YoutubeDL) wait() {
 func (r *YoutubeDL) Close() error {
 	if r != nil {
 		if r.cmd != nil {
-			r.cmd.Process.Kill()
+			r.cmd.Process.Kill() //nolint
```

Comment:
What does the lint complain over here?

Reply:
because there is an `error` output. but we don't need to check it there.